_source/logzio_collections/_prometheus-sources/telegraf-burrow.md

```diff
@@ -0,0 +1,92 @@
+---
```

Comment:
its for Collect Kafka topic, consumer and partition status
So i think it should be under kafka